research/fed-sm/virtualenv/min-requirements.txt

```diff
@@ -0,0 +1,7 @@
+nvflare>=2.1
```

Comment:
Is this the correct minimum requirement?

Reply:
changed to 2.3.0